DemoqaProject/tests/ElementsTest/RadioButtonTest.cs

```diff
@@ -0,0 +1,32 @@
+using DemoqaProject.PageObjects;
+using NUnit.Framework;
+
+namespace DemoqaProject
+{
+    public class RadioButtonTest : BaseSeleniumTest
+    {
+        [Test]
+        public void ClickAndCheck()
+        {
+            HomePage homePage = new HomePage();
+            homePage.NavigateToElementPage();
+            RadioButton radioButton = new RadioButton();
+            radioButton.NavigateToRadioButton();
+            
+            radioButton.CheckYesRadio();
+            Assert.That(radioButton.checkState.Text, Does.Contain("Yes"));
+            radioButton.CheckImpressiveRadio();
+            Assert.That(radioButton.checkState.Text, Does.Contain("Impressive"));
+        }
+
+        public void CheckNoButton()
+        {
+            HomePage homePage = new HomePage();
+            homePage.NavigateToElementPage();
+            RadioButton radioButton = new RadioButton();
+            radioButton.NavigateToRadioButton();
+            
+            Assert.IsTrue(radioButton.noRadio.Enabled);
```

Comment:
This test should check if that radio button is disabled for clicking. How it's enabled I don't know.

Reply:
The same, No idea why it is working in opposite side

Reply:
Try to check some specific attribute.

Reply:
Ok